simple/src/main/java/com/github/commonsrdf/simple/BlankNodeImpl.java

```diff
@@ -29,12 +27,11 @@
  */
 class BlankNodeImpl implements BlankNode {
 
-	private static AtomicLong bnodeCounter = new AtomicLong();
 	private final String id;
 	private final Optional<Graph> localScope;
 
 	public BlankNodeImpl() {
-		this(Optional.empty(), "b:" + bnodeCounter.incrementAndGet());
+		this(Optional.empty(), UUID.randomUUID().toString());
```

Comment:
Could you add the "b:" as a prefix back on so that it triggers the branch in ntriplesString?